Update getting started guide (#122)
spacetelescope/stenv: docs/source/getting_started.rst

    conda activate stenv

> (If you chose another name for the environment, substitute it for ``stenv``.)

Reviewer: ```suggestion
(If you chose another name when creating the environment, use that here instead.)
```

Activating a Conda environment changes which Python interpreter and packages are in use for that session (i.e. terminal window). Now, if you try to ``import jwst``::

    (stenv) $ python -c 'import jwst; print("ok")'

> Every time you open a new terminal window, you will need to activate the environment with ``conda activate stenv`` before you can use stenv software.

Reviewer: ```suggestion
Every time you open a new terminal window, you will need to activate the environment with ``conda activate <name>`` before you can use stenv software.

.. code-block:: shell
    conda activate stenv
```

.. note::
    You can show installed packages available within a Conda environment with ``conda list``.

> To deactivate an environment, run ``conda deactivate``:

Reviewer: ```suggestion
To deactivate an environment and return your shell to normal, run ``conda deactivate``:
```

    conda activate stenv

(If you chose another name when creating the environment, use that here instead.)

Activating a Conda environment changes which Python interpreter and packages are in use for that session (i.e. terminal window). Now, if you try to ``import jwst``::

    (stenv) $ python -c 'import jwst; print("ok")'

Every time you open a new terminal window, you will need to activate the environment with ``conda activate <name>`` before you can use stenv software.

> .. code-block:: shell
>     conda activate stenv

Reviewer: whoops, the docs build failed:
```
/home/docs/checkouts/readthedocs.org/user_builds/stenv/checkouts/122/docs/source/getting_started.rst:86: ERROR: Error in "code-block" directive:
maximum 1 argument(s) allowed, 4 supplied.
```
```suggestion
.. code-block:: shell

    conda activate stenv
```
